i18n/translations/en.json

```
@@ -372,9 +372,11 @@
   "more": "More",
   "name": "Name",
   "next": "Next",
+  "no": "No",
```

For consistency with the "on/off" strings:

```suggestion
  "no": "no",
```

---

Hmm I don't think we just think of this for the `toggle` field. What if we convert into lowercase in dialog? Would different languages be a problem?

---

Yeah, I don't think we can generalize this for all languages, so IMO we need to decide on one spelling that will be most useful for us. Lowercase "no/yes" would be consistent with the existing "off/on" toggles, but you are right that we might need the other spelling in other places.

I am split on this. @bastianallgeier @distantnative?

---

Let's keep it like it is. When we change it in the original source, they need to be changed for every other language in Transifex.

---

@bastianallgeier `no` and `yes` are new translation strings.

---

Oh, I thought we already had them. I only saw the diff. If on/off is lowercase, I would also go for lowercase. There are often no super elegant solutions with the translation strings that work everywhere.